app/lib/common/toolkit/buttons/danger_action_button.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:acter/common/themes/acter_theme.dart';
+import 'package:flutter/material.dart';
+import 'dart:math' as math;
+import 'dart:ui' show lerpDouble;
+
+class ActerDangerActionButton extends ElevatedButton {
```

Comment:
Do we require separate component as `ActerDangerActionButton` instead of using it same as `ActerPrimaryActionButton` with different button theme?

Reply:
IMHO, I think we do want this. First, because of semantic reasons: I don't understand why so many toolkits moved away from semantic naming things (like the famous original "Bootstrap" did) and instead went for purely function naming again, but I think it is way better to think of our toolkit in terms of higher level abstractions. So we want a special ActerDangerActionButton to be used whenever what is happening is potentially dangerous and the user should be warned about it. 

And that might move into more than just a difference in color: for e.g. for the nuke-button we have that really nice behavior that you have to _long hold_ the button to activate it and I am considering making that the default way to activate this button. To give the severity of the action the attention it deserves. And that would make this button not only look different but also behave different.

Last, but not least, we might decide to not use the ElevatedButton as the baseline but one more astray from the PrimaryButton. All of these things considered, I think making this an explicit type is better in terms of creating a proper toolkit.